Makefile

```diff
@@ -3,6 +3,7 @@
 ##################
 
 STANDARD_TEST_OPTIONS= -v -r --timeout=2h --keep-going --randomize-all --randomize-suites
+BUILD_DATE= $(shell date +'%Y%m%d')
```

Comment:
I think it's safer to use a build date has to be a day older.
>Images are published every day at 1am UTC
The tag uses the format YYYYMMDD , so the tag for 12th July 2023 would be 20230712

Ref: https://github.com/rancher/hosted-providers-e2e/actions/runs/7393592413/job/20113753836

Reply:
Build date that is a day older might not work since build is only preserved for 24 hours and then new one is generated.
The above run failed as GKE does not publishes nightly chart yet (only AKS, EKS)

Reply:
Oh, my bad. That's the only operator I tested with :sweat_smile: